specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json

```diff
@@ -1332,6 +1342,52 @@
         }
       ]
     },
+    "RedisCreateParameters": {
```

Comment:
Your create does not have `tags` anymore. On purpose?

Reply:
Nice catch. Fixed now.